lntest/node.go

```diff
@@ -331,6 +333,13 @@ func (hn *HarnessNode) start(lndError chan<- error) error {
 		hn.logFile = file
 	}
 
+	// we first apply delay config to the node
```

Comment:
maybe mention using "tcconfig we create a new delay shell" or whatever the right phrase is?